path_role.go

```diff
@@ -529,6 +538,14 @@ func (b *jwtAuthBackend) pathRoleCreateUpdate(ctx context.Context, req *logical.
 		role.AllowedRedirectURIs = allowedRedirectURIs.([]string)
 	}
 
+	callbackMode := data.Get("callback_mode").(string)
+	switch callbackMode {
+	case callbackModeDirect, callbackModeClient:
+		role.CallbackMode = callbackMode
+	default:
```

Comment:
This probably needs to check for empty string too.

Reply:
I don't think so.  I think this is only used if someone tries to configure and explicitly sets callbackmode.